.vscode/launch.json

```diff
@@ -29,6 +29,20 @@
             },
             "webRoot": "${workspaceRoot}"
         },
+        {
+            "type": "firefox",
```

Comment:
Had to install firefox debugger. Maybe it would be good to have a section in the README to describe on how to setup development environment, debug etc. (not in this patch)